lib/sql/query-insert.js

```diff
@@ -54,13 +51,7 @@ module.exports = class Insert extends RowStreamQuery {
     for (var x = 0; x < preppedData.length; ++x) {
       const row = []
       for (var col of insertSchema) {
-        if (col in preppedData[x]) {
-          row.push(
-            col in preppedData[x]
-              ? '$' + this.values.push(preppedData[x][col])
-              : 'DEFAULT'
-          )
-        }
+        row.push('$' + this.values.push(preppedData[x][col]))
```

Comment:
🔴 Since `create` takes an array of multiple object types it's possible that one object will have a prop while another doesn't — in which case we'll want to use `DEFAULT`.

```javascript
Book.objects.create([{
  title: 'a hat full of sky',
  rating: 5
}, {
  title: 'snow crash'
  // no rating, use DEFAULT
}])
```

should produce

```sql
INSERT INTO books (title, rating) values ($1, $2), ($3, DEFAULT);
```

We should def add a test for this.